docs/guides/commands/samples/post-execution/post-execution_basic.cs

```diff
@@ -1,11 +1,14 @@
+// Bad code!!!
 var result = await _commands.ExecuteAsync(context, argPos, _services);
 if (result.CommandError != null)
 	switch(result.CommandError)
 	{
 		case CommandError.BadArgCount:
-			await context.Channel.SendMessageAsync("Parameter count does not match any command's.");
+			await context.Channel.SendMessageAsync(
+				"Parameter count does not match any command's.");
```

Comment:
```suggestion
				"Parameter count does not match any commands.");
```

Reply:
`Commands'` makes sense though, since we are comparing the command parameter count against the command's parameter count.